specs/tables.md

```diff
@@ -38,64 +38,64 @@ Proved by the state circuit.
 
 Type sizes:
 
-> - **txID**: 24 bytes
-> - **address**: 160 bytes
-> - **callID**: 25 bytes
-> - **stackPointer**: 10 bytes
-> - **memoryAddress**: 40 bytes
+> - **txID**: 32 bits
+> - **address**: 160 bits
+> - **callID**: 32 bits
+> - **stackPointer**: 10 bits
+> - **memoryAddress**: 160 bits
```

Comment:
Even though the `memoryAddress` is assigned to a column which is 160 bits, I think it's worth specifying here that `memoryAddress` is 40 bits, or 32 bits actually, to follow the comment about maximum memory address size (so that we know the corresponding range constraint is guaranteed somewhere)

Reply:
I was actually thinking that we would "allow" memoryAddress to have up to 160 bits, knowing that that's actually impossible because you wouldn't be able to pay the gas cost.

I agree it's a better idea to add explicit constraints in the state circuit that it must fit into 32 bits.